service/src/main/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceImplHgvs.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2015 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.cbioportal.genome_nexus.service.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cbioportal.genome_nexus.service.*;
+import org.cbioportal.genome_nexus.service.annotation.NotationConverter;
+import org.cbioportal.genome_nexus.service.cached.CachedVariantAnnotationFetcher;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.*;
+
+@Service(value="VariantAnnotationServiceImplHgvs")
+public class VariantAnnotationServiceImplHgvs extends BaseVariantAnnotationServiceImpl
```

Comment:
I was wondering if `VariantAnnotationHgvsServiceImpl` or `HgvsVariantAnnotationServiceImpl` is better naming than `VariantAnnotationServiceImplHgvs`. Or, since we already have multiple implementations of the same service and using the `Qualifier`s to pick the right implementation, should we completely remove the `Impl` keyword? (https://octoperf.com/blog/2016/10/27/impl-classes-are-evil/)

Reply:
Yeah maybe we can just put it in the same folder n call it e.g. HgvsAnnotationService etc. 

Nice blog reference. We are using a lot of interfaces with a single implementation. Guess we could revisit that